systemd/systemd.go

```diff
@@ -490,6 +499,48 @@ func (s *systemd) daemonReloadNoLock() error {
 	return err
 }
 
+type ReloadReason struct {
```

Comment:
nitpick: when a type name contains reason, I generally expect it to be an enumeration or a string; here I think that something like `ReloadInitiators` would be better, or maybe just even adding a plural `ReloadReasons` is enough as a hint that this is not just a single fixed reason.